src/github.ts

```diff
@@ -112,6 +112,20 @@ export async function buildPiperFromSource (version: string): Promise<string> {
 
 export function getTag (version: string, forAPICall: boolean): string {
   version = version.toLowerCase()
-  if (version === '' || version === 'master' || version === 'latest') return 'latest'
+  if (version === '' || version === 'master' || version === 'latest') {
+    debug('Using latest tag')
+    return 'latest'
+  }
+  debug(`getTag returns: ${forAPICall ? 'tags' : 'tag'}/${version}`)
   return `${forAPICall ? 'tags' : 'tag'}/${version}`
 }
+
+export function getDownloadUrlByTag (version: string, forAPICall: boolean = false): string {
+  version = version.toLowerCase()
+  if (version === '' || version === 'master' || version === 'latest') {
```

Comment:
Do we still need to support `master` here or has that been replaced already upstream?

Reply:
Is it possible for some custom pipelines to use master/main as versions of Piper ?

Reply:
we don't have _master binaries anymore, but there might be pipelines where version is still 'master' and they might break if we just remove it. Some kind an anouncement should be made first